doc/getting_started/using_package_manager.rst

```diff
@@ -60,10 +60,10 @@ Here is how to create a simple test database after installation.
 
 #. Check if the default port the database instance will listen to is vacant.
 
-   Depending on the release, during installation Tarantool may start a
-   demonstrative global ``example.lua`` instance that listens to the
-   ``3301`` port by default. The ``example.lua`` file showcases basic
-   configuration and can be found in the ``/etc/tarantool/instances.enabled``
+   In versions before :doc:`2.4.2 </release/2.4.2>`, during installation
+   Tarantool may start a demonstrative global ``example.lua`` instance that
```

Comment:
"may start" — maybe we can be more specific as in the corresponding RN item: "... the Tarantool packages for Debian and Ubuntu automatically enable and start the ``example`` instance..." and so on. Otherwise, the condition when Tarantool may or may not start this ``example`` instance is not clear.
This suggestion is discussable because this system behavior is no longer the default one and we may not need such details.